python/lsst/ts/cbp/statemachine.py

```diff
@@ -0,0 +1,263 @@
+import logging
+from lsst.ts.cbp.component import CBPComponent
+import asyncio
+from salobj.base_csc import BaseCsc
+from salobj import State
+import SALPY_CBP
+
+
+class CBPCsc(BaseCsc):
+    """This defines the CBP CSC(Commandable SAL Component) using salobj
+
+    Parameters
+    ----------
+    port: str
+        This is the ip address of the CBP.
+
+    address: int
+        This is the port of the CBP
+
+    frequency: float
+        This is the amount of seconds to query the device for telemetry.
```

Comment:
This is ambiguous. Frequency is cycles/second which is how often to query, not for how long you try to query.